lib/demo/accounts/user_token.ex

```diff
@@ -0,0 +1,139 @@
+defmodule Demo.Accounts.UserToken do
+  use Ecto.Schema
+  import Ecto.Query
+
+  @hash_algorithm :sha256
+  @rand_size 32
+
+  # It is very important to keep the reset password token expiry short,
+  # since someone with access to the e-mail may take over the account.
+  @reset_password_validity_in_days 1
+  @confirm_validity_in_days 7
+  @change_email_validity_in_days 7
+  @session_validity_in_days 60
+
+  schema "users_tokens" do
```

Comment:
We changed it to “users_tokens” because it’s generated off the generators table_name param, which is plural. The table is essentially “#{table_name}_tokens”. It would have taken a significant amount of work to singularize the table name and didn’t seem worth it. You’re definitely welcome to change the table name after the generators run, though.